### DISCUSSION TITLE
name '_mysql' is not defined when add connection to mysql on MacOS

### DISCUSSION THREAD

**Discussion by zorro-gridi** [apache/airflow#23851] [General] [answered]
### Apache Airflow Provider(s)

mysql

### Apache Airflow version

2.3.0 (latest released)

### Operating System

macos

### Deployment

Virtualenv installation

### What happened

I'am using airflow to connect to my docker mysql container, but exception of "name '_mysql' is not defined" when testing the connection. I can get connected by using the Dbeaver.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Please - as advised. Do not open "airlfow issues" when what you want is troubleshoot your installation - use "discussions" instead. Converted into a discussion.

**Comment by potiuk** [comment, marked as answer]
Likely you have some mysql libraries missing in your system -but it's your deployment issue, not airflow issue. Please refer to sqlalchemy mysql documentation on the mysql drivers and their requirements.

**Reply by zorro-gridi** [reply]
I'am figure out that this is mysqlclient issue, and I have pymysql instead to use as the mysqlclient package then the issue has done. Thank you!
